CMakePresets.json

```diff
@@ -0,0 +1,234 @@
+{
+	"version": 7,
+	"cmakeMinimumRequired": {
+		"major": 3,
+		"minor": 27,
+		"patch": 0
+	},
+	"configurePresets": [
+		{
+			"name": "vcpkg",
+			"displayName": "Configure with vcpkg. please set VCPKG_ROOT environment first",
+			"toolchainFile": "$env{VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake"
+		},
+
+		{
+			"name": "default",
+			"displayName": "Default freerdp library",
+			"description": "Default freerdp library with vcpkg",
+			"inherits": "vcpkg",
+			"cacheVariables": {
+				"WITH_FFMPEG": "OFF",
+				"WITH_SWSCALE": "OFF",
+				"CHANNEL_URBDRC": "OFF",
+				"WITH_PKCS11": "OFF",
+				"WITH_CUPS": "OFF",
+				"WITH_MANPAGES": "OFF",
+				"WITH_KRB5": "OFF",
+				"WITH_CLIENT": "OFF",
+				"WITH_SERVER": "OFF",
+				"WITH_PROXY": "OFF"
+			}
+		},
+		{
+			"name": "default-msvc",
+			"displayName": "Default using Visual Studio 17 2022",
+			"inherits": "default",
+			"generator": "Visual Studio 17 2022",
+			"condition": {
+				"type": "equals",
+				"lhs": "${hostSystemName}",
+				"rhs": "Windows"
+			}
+		},
+		{
+			"name": "default-ninja",
+			"displayName": "Default using ninja",
+			"inherits": "default",
+			"generator": "Ninja"
+		},
+		{
+			"name": "default-android",
+			"displayName": "Default for android",
+			"hidden": true,
+			"inherits": "default",
+			"generator": "Unix Makefiles",
```

Comment:
no need for that, we're building android targets with `Ninja` as well

Reply:
This is for compiling Android on Windows